packages/plasma-ui/src/components/Pickers/TimePicker.tsx

```diff
@@ -40,14 +38,40 @@ const StyledDividers = styled.div`
     }
 `;
 
+/**
+ * Вернет массив чисел от `from` до `to` с интервалом `step`.
+ */
+const getRange = (from: number, to: number, step: number) =>
+    Array.from({ length: to - from + 1 }, (_, i) => from + i).filter((value) => value % step === 0);
```

Comment:
```suggestion
    const res = [];
    for (let i = from; i <= to; i += step) {
         res.push(i);
     }

     return res;
```

или оставим в одну строку?

Reply:
Бенч показывает результаты в пользу твоего кода https://jsbench.me/sxknobduy5/1